docs/site/content/docs/latest/getting-started-unmanaged.md

```diff
@@ -0,0 +1,80 @@
+<!-- markdownlint-disable MD036 -->
```

Comment:
The hugo build errors appear to be from these `markdownlint-disable` lines. I think I remember hitting something similar when trying to put these directives at the top of include files.

It appears hugo and mdlint are both happy if you move them down to follow the H1 title.

Reply:
addressed; thanks for figuring this out.